packages/react-events/src/Press.js

```diff
@@ -325,6 +340,22 @@ function calculateResponderRegion(target, props) {
   };
 }
 
+function getPointerType(nativeEvent: any) {
+  if (nativeEvent.pointerType != null) {
+    return nativeEvent.pointerType;
+  }
+  if (nativeEvent.type.indexOf('mouse') > -1) {
```

Comment:
Could `type` be pulled out to its own variable? Seems like a lot of property accesses each time otherwise and should reduce prod code size.